sass/_index.scss

```diff
@@ -1,335 +1,424 @@
-}
+        position: relative;
+        overflow: hidden;
 
-@keyframes Gradient {
-    0% {
-        background-position: 0% 50%
-    }
+        display: flex;
+        flex-direction: row;
+        @media (max-width: 767px) {
+            flex-direction: column;
+        }
+        justify-content: space-around;
 
-    50% {
-        background-position: 100% 50%
-    }
+        padding-bottom: 175px;
 
-    100% {
-        background-position: 0% 50%
-    }
-}
+        h2 {
+            color: #000;
+            font-weight: 700;
+            margin: auto 0 auto 5rem;
+            @media (max-width: 767px) {
+                font-size: 34px;
+                margin: 0 50px 50px 50px;
+            }
+            max-width: 28rem;
+        }
 
-.dark-bg {
-    background-color: #f4f4f4;
-}
+        img {
+            margin: auto 0px;
 
-.imagine {
-    font-size: 17px;
-    font-weight: normal;
-    line-height: 1.5em;
-    text-align: left;
-    margin-left: auto;
-    margin-right: auto;
-    max-width: 400px;
-
-    ul {
-        list-style: none;
-    }
+            width: 469px;
+            height: 418px;
 
-    li {
-        text-indent: -.8em;
-        margin-left: 0em;
-        margin-bottom: .5em;
-    }
+            @media (max-width: 767px) {
+                margin: 0px auto;
+                width: 323px;
+                height: 288px;
+            }
+        }
 
-    h1 {
-        text-align: start;
-        padding: 0em;
-    }
-}
+        #personal_bottom {
+            position: absolute;
+            bottom: -340px;
+            width: 150%;
+            height: 400px;
+            background-color: black;
+            border-radius: 100% 100%;
 
-.thisismatrix {
-    h1 {
-        text-align: center;
-        font-weight: bold;
-        padding: 0em;
+            left: 50%;
+            transform: translateX(-50%);
+
+            @media (max-width: 767px) {
+                width: 200%;
+            }
+        }
     }
 
-    p {
-        max-width: 640px;
-        margin-left: auto;
-        margin-right: auto;
+    #use-case-community {
+        background-color: #000;
+
+        display: flex;
+        flex-direction: column;
+        justify-content: space-around;
+
         text-align: center;
-    }
-}
+        padding-top: 60px;
+        padding-bottom: 100px;
 
-.howdoesitwork {
-    text-align: center;
-    padding-top: 20px;
-    padding-bottom: 20px;
+        h2 {
+            color: #fff;
+            font-size: 40px;
+            font-weight: 700;
+            max-width: 36rem;
+            margin: 0 auto;
+
+            @media (max-width: 767px) {
+                font-size: 34px;
+                margin: 0 50px 50px 50px;
+            }
+        }
 
-    h2 {
-        margin-top: 20px;
-        margin-bottom: 20px;
-    }
+        .call-to-action {
+            background-color: #fff;
+            color: #000;
+            font-size: 18px;
+            font-weight: 600;
+            border-radius: 1000px;
```

Comment:
Is this making an element round? I think you can do `50%` instead.

Reply:
Using `50%` doesn't seem to produce the expected result. It makes the whole div an ellipse, instead of just rounding the corners.

Reply:
Okay, nevermind then.